va/validation-authority_test.go

```diff
@@ -507,9 +569,12 @@ func TestCAAChecking(t *testing.T) {
 	}
 
 	present, valid, err := va.CheckCAARecords(core.AcmeIdentifier{Type: "dns", Value: "dnssec-failed.org"})
-	test.AssertError(t, err, "dnssec-failed.org")
-	test.Assert(t, !present, "Present should be false")
-	test.Assert(t, !valid, "Valid should be false")
+	// TODO(Issue #401): Until #401 is resolved ignore DNS timeouts from non-local resolver
+	if err == nil || err != nil && err.Error() != "read udp 8.8.8.8:53: i/o timeout" {
+		test.AssertError(t, err, "dnssec-failed.org")
```

Comment:
Reliance on external resources is not OK.  Please add a comment referring to an issue to fix this.